### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by joelea**

### CODE REVIEW

**Comment by joelea** [comment]
@jboreiko @gbrova - this line up with what we talked about?

@clockfort @rjullman @tmgordeeva @nemines @tpetracca @schlosna In our London retrospective today we talked about the fact that PRs can take a long time to merge. This was one idea for making sure they merge faster - making sure every PR has an assignee who is responsible for reviewing or finding a new reviewer/adding other people who need to review. Are people ok to try this out?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- [ ] Someone has been assigned to review this PR
```
Comment (joelea) [inline .github/PULL_REQUEST_TEMPLATE.md]: Bet viewed as rendered markdown to get what we're going for.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+**Reviewer** (@joelea if unsure):
```
Comment (jboreiko) [inline .github/PULL_REQUEST_TEMPLATE.md]: Still a danger that this becomes a list of people, none of which feel the obligation to move it forward.  Would vote for Primary Reviewer.

**Comment by clockfort** [comment]
I would CR this, but no one is marked as assignee :stuck_out_tongue: 

I think our side of things agrees with this and it's what we do already in gerrit.

**Pull request merged**